<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>SolidDistributionsEditor</class>
  <widget class="QWidget" name="SolidDistributionsEditor">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>973</width>
        <height>559</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Form</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>0</number>
      </property>
      <property name="rightMargin">
        <number>0</number>
      </property>
      <property name="bottomMargin">
        <number>0</number>
      </property>
      <item>
        <widget class="QFrame" name="frameAllElements">
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,7">
            <property name="leftMargin">
              <number>0</number>
            </property>
            <property name="topMargin">
              <number>0</number>
            </property>
            <property name="rightMargin">
              <number>0</number>
            </property>
            <property name="bottomMargin">
              <number>0</number>
            </property>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
                <item>
                  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,2">
                    <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                          <widget class="QLabel" name="label_4">
                            <property name="text">
                              <string>Selected time point [s]:</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                            <property name="margin">
                              <number>0</number>
                            </property>
                            <property name="indent">
                              <number>5</number>
                            </property>
                            <property name="toolTip">
                              <string>Selected time point</string>
                            </property>
                            
                            <property name="whatsThis">
                              <string>Selected time point</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QLabel" name="labelTime">
                            <property name="minimumSize">
                              <size>
                                <width>30</width>
                                <height>0</height>
                              </size>
                            </property>
                            <property name="text">
                              <string>0</string>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                            <property name="toolTip">
                              <string>Selected time point</string>
                            </property>
                            
                            <property name="whatsThis">
                              <string>Selected time point</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <spacer name="horizontalSpacer_4">
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                              <size>
                                <width>40</width>
                                <height>20</height>
                              </size>
                            </property>
                          </spacer>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <widget class="CQtTable" name="tableWidgetCompoundsFractions">
                        <property name="toolTip">
                          <string>Mass fraction (0..1) of each defined compound in the solid phase at the corresponding time point</string>
                        </property>
                        
                        <property name="whatsThis">
                          <string>Mass fraction (0..1) of each defined compound in the solid phase at the corresponding time point</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QListWidget" name="listWidget" />
                    </item>
                  </layout>
                </item>
                <item>
                  <widget class="QTableWidget" name="tableWidgetDistributions">
                    <property name="enabled">
                      <bool>false</bool>
                    </property>
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="editTriggers">
                      <set>QAbstractItemView::NoEditTriggers</set>
                    </property>
                    <property name="tabKeyNavigation">
                      <bool>false</bool>
                    </property>
                    <property name="alternatingRowColors">
                      <bool>false</bool>
                    </property>
                    <property name="selectionMode">
                      <enum>QAbstractItemView::NoSelection</enum>
                    </property>
                    <attribute name="horizontalHeaderCascadingSectionResizes">
                      <bool>false</bool>
                    </attribute>
                    <attribute name="horizontalHeaderStretchLastSection">
                      <bool>true</bool>
                    </attribute>
                    <attribute name="verticalHeaderCascadingSectionResizes">
                      <bool>false</bool>
                    </attribute>
                    <attribute name="verticalHeaderDefaultSectionSize">
                      <number>19</number>
                    </attribute>
                    <attribute name="verticalHeaderStretchLastSection">
                      <bool>false</bool>
                    </attribute>
                    <column>
                      <property name="text">
                        <string>Distributions sequence</string>
                      </property>
                    </column>
                    <property name="toolTip">
                      <string>List of defined distributed properties</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>List of defined distributed properties</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QFrame" name="frameDistributions">
                <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="leftMargin">
                    <number>0</number>
                  </property>
                  <property name="topMargin">
                    <number>0</number>
                  </property>
                  <property name="rightMargin">
                    <number>0</number>
                  </property>
                  <property name="bottomMargin">
                    <number>0</number>
                  </property>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                        <widget class="QLabel" name="label_5">
                          <property name="text">
                            <string>Compound:</string>
                          </property>
                          <property name="toolTip">
                            <string>Compound or total mixture to set distributed properties</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Compound or total mixture to set distributed properties</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QComboBox" name="comboBoxCompound">
                          <property name="minimumSize">
                            <size>
                              <width>90</width>
                              <height>0</height>
                            </size>
                          </property>
                          <property name="toolTip">
                            <string>Compound or total mixture to set distributed properties</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Compound or total mixture to set distributed properties</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QLabel" name="labelRows">
                          <property name="text">
                            <string>Rows:</string>
                          </property>
                          <property name="toolTip">
                            <string>Distributed property to edit</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Distributed property to edit</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QComboBox" name="comboBoxRows">
                          <property name="minimumSize">
                            <size>
                              <width>90</width>
                              <height>0</height>
                            </size>
                          </property>
                          <property name="toolTip">
                            <string>Distributed property to edit</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Distributed property to edit</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QLabel" name="labelColumns">
                          <property name="text">
                            <string>Columns:</string>
                          </property>
                          <property name="toolTip">
                            <string>Distributed property to edit</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Distributed property to edit</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QComboBox" name="comboBoxColumns">
                          <property name="minimumSize">
                            <size>
                              <width>90</width>
                              <height>0</height>
                            </size>
                          </property>
                          <property name="toolTip">
                            <string>Distributed property to edit</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Distributed property to edit</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QLabel" name="labelPSDType">
                          <property name="text">
                            <string>PSD type:</string>
                          </property>
                          <property name="toolTip">
                            <string>Additional settings for PSD: PSD type</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Additional settings for PSD: PSD type</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QComboBox" name="comboBoxPSDType">
                          <property name="minimumSize">
                            <size>
                              <width>90</width>
                              <height>0</height>
                            </size>
                          </property>
                          <property name="toolTip">
                            <string>Additional settings for PSD: PSD type</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Additional settings for PSD: PSD type</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QLabel" name="labelPSDGridType">
                          <property name="text">
                            <string>PSD grid:</string>
                          </property>
                          <property name="toolTip">
                            <string>Additional settings for PSD: PSD grid</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Additional settings for PSD: PSD grid</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QComboBox" name="comboBoxPSDGridType">
                          <property name="minimumSize">
                            <size>
                              <width>90</width>
                              <height>0</height>
                            </size>
                          </property>
                          <property name="toolTip">
                            <string>Additional settings for PSD: PSD grid</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Additional settings for PSD: PSD grid</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <spacer name="horizontalSpacer">
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                            <size>
                              <width>40</width>
                              <height>20</height>
                            </size>
                          </property>
                        </spacer>
                      </item>
                    </layout>
                  </item>
                  <item>
                    <widget class="CQtTable" name="tableWidget">
                      <property name="toolTip">
                        <string>Values of the distributed property</string>
                      </property>
                      
                      <property name="whatsThis">
                        <string>Values of the distributed property</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                        <widget class="QPushButton" name="pushButtonFunctional">
                          <property name="enabled">
                            <bool>false</bool>
                          </property>
                          <property name="text">
                            <string>Insert distribution</string>
                          </property>
                          <property name="toolTip">
                            <string>Insert functional distribution</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Insert functional distribution</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <spacer name="horizontalSpacer_2">
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                            <size>
                              <width>40</width>
                              <height>20</height>
                            </size>
                          </property>
                        </spacer>
                      </item>
                      <item>
                        <widget class="QPushButton" name="pushButtonApply">
                          <property name="enabled">
                            <bool>false</bool>
                          </property>
                          <property name="text">
                            <string>Apply</string>
                          </property>
                          <property name="toolTip">
                            <string>Apply all changes made to distributed parameters</string>
                          </property>
                          
                          <property name="whatsThis">
                            <string>Apply all changes made to distributed parameters</string>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                </layout>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>CQtTable</class>
      <extends>QTableWidget</extends>
      <header location="global">QtTable.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>tableWidgetCompoundsFractions</tabstop>
    <tabstop>listWidget</tabstop>
    <tabstop>tableWidgetDistributions</tabstop>
    <tabstop>comboBoxCompound</tabstop>
    <tabstop>comboBoxRows</tabstop>
    <tabstop>comboBoxColumns</tabstop>
    <tabstop>comboBoxPSDType</tabstop>
    <tabstop>comboBoxPSDGridType</tabstop>
    <tabstop>tableWidget</tabstop>
    <tabstop>pushButtonFunctional</tabstop>
    <tabstop>pushButtonApply</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>